src/PrestaShopBundle/Form/Admin/Type/CustomMoneyType.php

```diff
@@ -55,4 +84,31 @@ public function configureOptions(OptionsResolver $resolver)
 
         $resolver->setAllowedTypes('scale', 'int');
     }
+
+    /**
+     * {@inheritDoc}
+     */
+    public function buildView(FormView $view, FormInterface $form, array $options)
+    {
+        parent::buildView($view, $form, $options);
+        $currency = $options['currency'];
+        $moneyPattern = $view->vars['money_pattern'];
+
+        $priceSpecification = $this->locale->getPriceSpecification($currency);
+        $value = new DecimalNumber((string) (float) $view->vars['value']);
+        $pricePattern = $value->isPositive() ? $priceSpecification->getPositivePattern() : $priceSpecification->getNegativePattern();
+
+        // inspired by symfony MoneyType
+        preg_match('/^([^\s\xc2\xa0]*)[\s\xc2\xa0]*{{ widget }}(?:[,.]0+)?[\s\xc2\xa0]*([^\s\xc2\xa0]*)$/u', $moneyPattern, $matches);
```

Comment:
find currency symbol in string (find all except `{{ widget }}` which is the way how symfony MoneyType formats it). preg_match taken from MoneyType ($matches[1] should contain symbol if it exists)